web/react/utils/utils.jsx

```diff
@@ -1221,3 +1221,7 @@ export function getPostTerm(post) {
 
     return postTerm;
 }
+
+export function featureToggle(feature) {
+    return PreferenceStore.getPreference(Constants.Preferences.CATEGORY_ADVANCED_SETTINGS, Constants.FeatureTogglePrefix + feature, {value: 'false'}).value === 'true';
```

Comment:
This function sounds like it's actually toggling the state of the feature when it only checks the current state. Maybe `featureEnabled()` would be a better name?